src/frontend/src/eth/components/fee/FeeDisplay.svelte

```diff
@@ -41,19 +46,37 @@
 
 		clearTimeout(timer);
 	});
+
+	const balance = nonNullish($feeIdStore)
```

Comment:
We don't have the use case yet, but as a precaution, shouldn't we set the default value to `undefined` to ensure nothing is displayed until the balance is defined? This way, we can avoid any flickering effect, such as "display, not display, display again."

Reply:
right! agreed! changed to `undefined` as default